doc/changes/changelog.md

```diff
@@ -0,0 +1 @@
+changes_0.3.0.md
```

Comment:
Let's make this file unified: 
https://raw.githubusercontent.com/exasol/sql-statement-builder/master/doc/changes/changelog.md

Reply:
ah, ok. There I missunderstood the template and thought it should be a symlink.

> Link from doc/changes/changelog.md to new log entry file

By the way, in the template it says, we should use changes_ with an underscore. In the link above, you used and `-`